_inc/client/plans/single-product-backup.jsx

```diff
@@ -29,13 +36,30 @@ export function SingleProductBackup( { plan, products, upgradeLinks, isFetchingD
 				{ isFetchingData ? (
 					<div className="plans-section__single-product-skeleton is-placeholder" />
 				) : (
-					<SingleProductBackupCard products={ products } upgradeLinks={ upgradeLinks } />
+					<SingleProductBackupCard
+						products={ products }
+						upgradeLinks={ upgradeLinks }
+						selectedBackupType={ selectedBackupType }
+						setSelectedBackupType={ setSelectedBackupType }
+					/>
 				) }
 			</div>
 		</React.Fragment>
 	);
 }
 
+function PromoNudge() {
+	return (
+		<div className="single-product-backup__promo">
+			<div className="single-product-backup__promo-star">{ __( 'Up to 70% off!' ) }</div>
+			<h4 className="single-product-backup__promo-header">
+				{ __( 'Hurry, these are' ) }
+				<strong>{ __( 'Limited time introductory prices!' ) }</strong>
```

Comment:
This is going to be problematic for translators. I think it would be best to have this be one string only.

Reply:
Combined in 4fc316f6607456a9ab22642a9d001203dd4d7b01